elftools/dwarf/die.py

```diff
@@ -237,11 +237,18 @@ def _parse_DIE(self):
 
         # Guided by the attributes listed in the abbreviation declaration, parse
         # values from the stream.
-        for name, form in abbrev_decl.iter_attr_specs():
+        for spec in abbrev_decl['attr_spec']:
```

Comment:
Because we may need the third attribute (value) from the attr spec, when the form is implicit const.
I didn't like changing the method to return an almost useless value in most cases as a third member in the returned tuple, or changing it's behaviour completely to not return a tuple anymore.

But this is probably not ideal. If you think of a better way of handling this without changing the whole interface, let me know !